gaas-ui/src/util/sanitize-inputs.js

```diff
@@ -0,0 +1,21 @@
+export default function sanitizeInputs(string) {
```

Comment:
just realised the name of this is misleading, renamed it to escapingInputs. We do have sanitising input libraries we're using, which removes unwanted JS for example, this just escapes the characters instead.

Reply:
I am just not sure a custom util is needed, libraries like [DOMPurify](https://github.com/cure53/DOMPurify) will do this for you very reliably. Just an example I am sure there are others. This can also be used to sanitize urls. Also worth mentioning, you list `sanitize-html` in the package.json but don't use it.

Reply:
removed and replaced with DOMPurify.

Reply:
<img width="384" alt="Screenshot 2022-03-21 at 10 45 01 am" src="https://user-images.githubusercontent.com/60354187/159246172-e0d00610-545d-427f-b6c0-177896e30675.png">
<img width="375" alt="Screenshot 2022-03-21 at 10 45 07 am" src="https://user-images.githubusercontent.com/60354187/159246176-ff0181c0-d0a5-4053-be01-51c9098d73f8.png">

DOMPurify doesn't seem to sanitise img or b tags. I've tested it with script and body tags and that gets sanitised.

Reply:
I'm looking for a HTML Encode library to deal with the unacceptable characters.

Reply:
Using the encode function from [html-entities](https://www.npmjs.com/package/html-entities) to encode unacceptable characters. All the inputs have been refactored to use this. I've left in the DOMPurify library too for all the inputs so that it deals with keyword specific inputs.